File: algorithms/python3/checkIfNAndItsDoubleExist.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        if (arr == None or len(arr) == 0):
            return False
        for i in range(len(arr)):
            for j in range(len(arr)):
                if (i != j and arr[i] == 2 * arr[j] and 0 <= i and j < len(arr)):
                    return True
        return False

File: algorithms/c/checkIfNAndItsDoubleExist.c
bool checkIfExist(int* arr, int arrSize){
    if (arr == NULL || arrSize == 0) {
        return false;
    }
    for (int i = 0; i < arrSize; i++) {
        for (int j = 0; j < arrSize; j++) {
            if (arr[i] != arr[j] && arr[i] == 2 * arr[j] && 0 <= i && j < arrSize) {
                return true;
            }
        }
    }
    return false;
}
